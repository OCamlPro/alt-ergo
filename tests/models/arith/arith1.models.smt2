(set-logic ALL)
(set-option :produce-models true)
(declare-const x Int)
(assert (<= x 42))
(check-sat)
(get-model)
