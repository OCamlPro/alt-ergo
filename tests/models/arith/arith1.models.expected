
unknown
(
  (define-fun x () Int 0)
)
